fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerTest.java

```diff
@@ -2648,7 +2648,7 @@ public void testConfigLoader() throws Exception {
         try (OnlineIndexer indexBuilder = OnlineIndexer.newBuilder()
                 .setDatabase(fdb).setMetaData(metaData).setIndex(index).setSubspace(subspace)
                 .setConfigLoader(old ->
-                        OnlineIndexer.Config.newBuilder()
+                        old.toBuilder()
```

Comment:
Could this test check `increaseLimitAfter` and `progressLogIntervalMillis` being unchanged after loading configs?

Reply:
Ah, I misread your earlier comment. Added asserts for them too.